codegen/cmd/injection-gen/generators/informer.go

```diff
@@ -104,17 +149,23 @@ func (g *injectionGenerator) GenerateType(c *generator.Context, t *types.Type, w
 var injectionInformer = `
 func init() {
 	{{.injectionRegisterInformer|raw}}(withInformer)
+	{{.injectionRegisterDynamicInformer|raw}}(withDynamicInformer)
 }
 
 // Key is used for associating the Informer inside the context.Context.
 type Key struct{}
 
 func withInformer(ctx {{.contextContext|raw}}) ({{.contextContext|raw}}, {{.controllerInformer|raw}}) {
 	f := {{.factoryGet|raw}}(ctx)
-	inf := f.{{.group}}().{{.version}}().{{.type|publicPlural}}()
+	inf := f.{{.groupGoName}}().{{.versionGoName}}().{{.type|publicPlural}}()
 	return context.WithValue(ctx, Key{}, inf), inf.Informer()
 }
 
+func withDynamicInformer(ctx {{.contextContext|raw}}) {{.contextContext|raw}} {
+	inf := &wrapper{client: {{ .clientGet|raw }}(ctx)}
+	return context.WithValue(ctx, Key{}, inf)
```

Comment:
here again, `context.WithValue` should come from a registered type

Reply:
Done